Assets/MixedRealityToolkit/Utilities/Gltf/Schema/GltfInterpolationType.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+namespace Microsoft.MixedReality.Toolkit.Utilities.Gltf.Schema
+{
+    /// <summary>
+    /// Interpolation algorithm.
+    /// https://github.com/KhronosGroup/glTF/blob/master/specification/2.0/schema/animation.sampler.schema.json
+    /// </summary>
+    public enum GltfInterpolationType
+    {
+        LINEAR,
```

Comment:
Can we add summary tags? Also, our coding convention does not use all caps names.

Reply:
Synced offline: url should be enough, most enums don't have summary stubs from khronos gltf standard schema declaration.